jenkins/common.groovy

```diff
@@ -131,6 +131,39 @@ def setup() {
   }
 }
 
+def checkForBrokenLinks(String buildLogPath, String envName) {
+  // Look for broken links
+  def csvFileName = "${envName}-broken-links.csv" // For use within the docker container
+  def csvFile = "${WORKSPACE}/vets-website/${csvFileName}" // For use outside of the docker context
+
+  // Ensure the file isn't there if we had to rebuild
+  if (fileExists(csvFile)) {
+    sh "rm /application/${csvFileName}"
+  }
+
+  // Output a csv file with the broken links
+  sh "cd /application && jenkins/glean-broken-links.sh ${buildLogPath} ${csvFileName}"
+  if (fileExists(csvFile)) {
+    echo "Found broken links; notifying the Slack channel."
+    // TODO: Move this slackUploadFile to cacheDrupalContent and update the echo statement above
+    // slackUploadFile(filePath: csvFile, channel: 'dev_null', failOnError: true, initialComment: "Found broken links in the ${envName} build on `${env.BRANCH_NAME}`.")
+
+    // Until slackUploadFile works...
+    def linkCount = sh(returnStdout: true, script: "cd /application && wc -l ${csvFileName} | cut -d ' ' -f1") as Integer
+    slackSend message: "@here ${linkCount - 1} broken links found in the ${envName} build on `${env.BRANCH_NAME}`\n${env.RUN_DISPLAY_URL}".stripMargin(),
+      color: 'danger',
+      failOnError: true,
+      channel: 'cms-general'
```

Comment:
It doesn't look like the CSV is attached to the Slack message, so does that mean that each time this happens, content will need to coordinate with devs to figure out which links are bad?

Reply:
You're exactly right. I was hoping to get that part working in this PR, but it turns out that [`slackUploadFile` command](https://jenkins.io/doc/pipeline/steps/slack/) isn't available. I don't know what to do about that right now, but I'll be writing up another ticket to address that part. In the meantime, this is the MVP.